examples/Console/TestMetrics.cs

```diff
@@ -70,6 +71,16 @@ internal static object Run(MetricsOptions options)
                         o.IsDelta = options.IsDelta;
                     });
             }
+            else if (options.UseExporter.ToLower() == "inmemory")
```

Comment:
i'll probably leave this examples untouched. They are usually used for validating something visually (like in console or jaeger/zipkin etc).
InMemory is used for unittests typically. So we may use it in Unittests (which acts as usage example)

Reply:
I was also not sure if this was the right thing to do. I checked the existing `InMemoryExporter` example for reference and added it here.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/Console/TestInMemoryExporter.cs#L40

Reply:
I have removed the changes.